.ci/Jenkinsfile

```diff
@@ -61,6 +60,7 @@ pipeline {
                   unstash 'source'
                   dir(BASE_DIR){
                     sh script: """.ci/scripts/install-dependencies.sh "${GO_VERSION}" """, label: 'Install dependencies'
+                    sh 'ls -ltrah && pwd && env | sort'
```

Comment:
Do we want `env` to be executed on CI?

Reply:
debugging yet as it works locally but not in the CI

```
14:39:51  hookid: golangci-lint
14:39:51  
14:39:51  level=error msg="Running error: context loading failed: no go files to analyze"
```